addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -217,6 +217,9 @@ options.registry.gallery = options.Class.extend({
      * @see this.selectClass for parameters
      */
     mode: function (previewMode, widgetValue, params) {
+        if (this.options.wysiwyg) {
+            this.options.wysiwyg.odooEditor.unbreakableStepUnactive();
+        }
```

Comment:
- Are you not supposed to re-enable it at the end of the function ?
- Maybe also disable it for the smallest code as possible (so maybe not at the start of the method but before the action that is not working?
- Add a comment to justify the need for this
- Mention in the commit message that this bug is only on Chrome :)

Reply:
> * Are you not supposed to re-enable it at the end of the function ?

By checking the code on `OdooEditor`, I saw that the code of `historyStep()` should automatically enable the unbreakable check for the next one (it sets the [_checkStepUnbreakable](https://github.com/odoo/odoo/blob/16.0/addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js#L1105) and [_toRollback](https://github.com/odoo/odoo/blob/16.0/addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js#L1083) to `false`). I also checked in the codebase, and it's only disabled before the targeted action :+1:

> * Maybe also disable it for the smallest code as possible (so maybe not at the start of the method but before the action that is not working?

Moved right before `this[widgetValue]();` which applies the mode changes :+1

> * Add a comment to justify the need for this

:+1: 

> * Mention in the commit message that this bug is only on Chrome :)

:+1: 

**Remark:** while re-testing this code, I found a strange behaviour (related to Chrome too):

With chrome cache enabled, current images on `masonry` layout are not loaded correctly and stay invisible after the update (when I disable chrome cache, layout adaptation works as intended):

https://user-images.githubusercontent.com/25032876/232032275-d0bdc805-e04b-4a92-8e4f-a416d3e2cfee.mp4

The simplest fix here was to use a "clone" of the image to force the loading... (I added this in a separate commit, since not linked to the history issue :+1:)